addons/l10n_vn/__init__.py

```diff
@@ -3,6 +3,13 @@
 
 # This module is Copyright (c) 2009-2013 General Solutions (http://gscom.vn) All Rights Reserved.
 
+from odoo import api, SUPERUSER_ID
+
+
+def _post_init_hook(cr, registry):
+    _preserve_tag_on_taxes(cr, registry)
```

Comment:
@hpl-odoo @jva-odoo I don't think this preserve_tag_on_taxes is necessary.  Is there any reason for this.

Reply:
@jco-odoo  previously, `_preserve_tag_on_taxes` called directly from `post_init_hook` and since VN is now multilingual and thus `process_coa_translations` needs to be called from `post_init_hook` again. So, we just added a parent method that calls both of them. Do you think we can remove `_preserve_tag_on_taxes` all-together from master?

Reply:
I don't see why we need it here.  It seems like an old relic from the past.  Can we remove this line here?

Reply:
ok, I will remove it.